[IMP]Modified the method exchange_rate

rv-clearcorp opened the pull request.

Commit:
[IMP]Modified the method exchange_rate

Commit:
[FIX]repare some bugs

Commit:
[FIX]repair bugs

ghost commented:
Delete method _currency_priority.

In the method get_exchange_rate (res.currency), you need the date in context to compute the rate (_current_rate)

Commit:
[FIX]repair bugs

Delete method _currency_priority.
In the method get_exchange_rate (res.currency), add the context to
compute the rate (_current_rate)

Commit:
[FIX] Remove invalid comment in ccorp_project_scrum
